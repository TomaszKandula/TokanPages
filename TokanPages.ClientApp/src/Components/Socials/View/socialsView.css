.socials-card-image-holder {
    margin: 0;
    border-style: solid;
    border-width: 4px;
    border-color: #6367ef;
    border-radius: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
    overflow: hidden;
    position: absolute;
    z-index: 999;
    height: 88px;
    width: 88px;
}

.socials-avatar {
    height: 80px;
    width: 80px;
}

.socials-image {
    height: 156px !important;
    object-fit: cover;
}
