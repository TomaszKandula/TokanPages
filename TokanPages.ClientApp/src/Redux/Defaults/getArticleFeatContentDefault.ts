import { IGetArticleFeatContent } from "../../Redux/States/getArticleFeatContentState";

export const GetArticleFeatContentDefault: IGetArticleFeatContent = 
{
    isLoading: false,
    content: 
    {
        title: "",
        desc: "",
        text1: "",
        text2: "",
        button: "",
        image1: "",
        image2: "",
        image3: "",
        image4: ""
    }
}