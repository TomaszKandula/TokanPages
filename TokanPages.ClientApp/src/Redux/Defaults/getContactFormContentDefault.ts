import { IGetContactFormContent } from "Redux/States/getContactFormContentState";

export const GetContactFormContentDefault: IGetContactFormContent = 
{
    isLoading: false,
    content: 
    {
        caption: "",
        text: "",
        button: ""
    }
}