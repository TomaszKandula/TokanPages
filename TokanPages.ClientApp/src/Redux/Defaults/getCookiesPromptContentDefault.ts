import { IGetCookiesPromptContent } from "../States/getCookiesPromptContentState";

export const GetCookiesPromptContentDefault: IGetCookiesPromptContent = 
{
    isLoading: false,
    content: 
    {
        caption: "",
        text: "",
        button: "",
        days: 0
    }
}