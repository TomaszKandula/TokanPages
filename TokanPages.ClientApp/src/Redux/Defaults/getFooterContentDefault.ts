import { IGetFooterContent } from "../States/getFooterContentState";

export const GetFooterContentDefault: IGetFooterContent = 
{
    isLoading: false,
    content: 
    {
        terms: "",
        policy: "",
        copyright: "",
        reserved: "",
        icons: []
   }    
}