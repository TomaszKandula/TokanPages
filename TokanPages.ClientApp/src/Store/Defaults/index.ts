export { ApplicationDialog } from "./Application/applicationDialog";
export { ApplicationError } from "./Application/applicationError";
export { ApplicationLanguage } from "./Application/applicationLanguage";
export { ApplicationMessage } from "./Application/applicationMessage";

export { ListArticlesDefault } from "./Articles/listArticlesDefault";
export { SelectArticleDefault } from "./Articles/selectArticleDefault";
export { UpdateArticleDefault } from "./Articles/updateArticleDefault";

export { GetAccountContentDefault } from "./Content/getAccountContentDefault";
export { GetActivateAccountContentDefault } from "./Content/getActivateAccountContentDefault";
export { GetArticleFeatContentDefault } from "./Content/getArticleFeaturesContentDefault";
export { GetClientsContentDefault } from "./Content/getClientsContentDefault";
export { GetContactFormContentDefault } from "./Content/getContactFormContentDefault";
export { GetCookiesPromptContentDefault } from "./Content/getCookiesPromptContentDefault";
export { GetFeaturedContentDefault } from "./Content/getFeaturedContentDefault";
export { GetFeaturesContentDefault } from "./Content/getFeaturesContentDefault";
export { GetFooterContentDefault } from "./Content/getFooterContentDefault";
export { GetHeaderContentDefault } from "./Content/getHeaderContentDefault";
export { GetNavigationContentDefault } from "./Content/getNavigationContentDefault";
export { GetNewsletterContentDefault } from "./Content/getNewsletterContentDefault";
export { GetPolicyContentDefault } from "./Content/getPolicyContentDefault";
export { GetResetPasswordContentDefault } from "./Content/getResetPasswordContentDefault";
export { GetStoryContentDefault } from "./Content/getStoryContentDefault";
export { GetTermsContentDefault } from "./Content/getTermsContentDefault";
export { GetTestimonialsContentDefault } from "./Content/getTestimonialsContentDefault";
export { GetUnsubscribeContentDefault } from "./Content/getUnsubscribeContentDefault";
export { GetUpdatePasswordContentDefault } from "./Content/getUpdatePasswordContentDefault";
export { GetUpdateSubscriberContentDefault } from "./Content/getUpdateSubscriberContentDefault";
export { GetUserSigninContentDefault } from "./Content/getUserSigninContentDefault";
export { GetUserSignoutContentDefault } from "./Content/getUserSignoutContentDefault";
export { GetUserSignupContentDefault } from "./Content/getUserSignupContentDefault";
export { GetWrongPagePromptContentDefault } from "./Content/getWrongPagePromptContentDefault";

export { AddSubscriberDefault } from "./Subscribers/addSubscriberDefault";
export { RemoveSubscriberDefault } from "./Subscribers/removeSubscriberDefault";
export { UpdateSubscriberDefault } from "./Subscribers/updateSubscriberDefault";

export { ActivateAccountDefault } from "./Users/activateAccountDefault";
export { ReAuthenticateUserDefault } from "./Users/reAuthenticateUserDefault";
export { RemoveAccountDefault } from "./Users/removeAccountDefault";
export { ResetUserPasswordDefault } from "./Users/resetUserPasswordDefault";
export { SigninUserDefault } from "./Users/signinUserDefault";
export { SignupUserDefault } from "./Users/signupUserDefault";
export { StoreUserDataDefault } from "./Users/storeUserDataDefault";
export { UpdateUserDefault } from "./Users/updateUserDefault";
export { UpdateUserPasswordDefault } from "./Users/updateUserPasswordDefault";
