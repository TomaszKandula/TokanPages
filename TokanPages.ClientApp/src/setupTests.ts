// jest-dom adds custom jest matchers for asserting on DOM nodes.
// allows you to do things like:
// expect(element).toHaveTextContent(/react/i)
// learn more: https://github.com/testing-library/jest-dom
import "@testing-library/jest-dom";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import enableHooks from "jest-react-hooks-shallow";

enableHooks(jest);

jest.mock("react-redux", () => (
{
    useSelector: jest.fn(),
    useDispatch: jest.fn()
}));

jest.mock("connected-react-router", () => (
{
    connectRouter: jest.fn(),
    routerMiddleware: jest.fn()
}));

Enzyme.configure({ adapter: new Adapter(), disableLifecycleMethods: false });

// environmental variables used during all tests
process.env.REACT_APP_API_VER="1";
process.env.REACT_APP_BACKEND="http://localhost:5000";
