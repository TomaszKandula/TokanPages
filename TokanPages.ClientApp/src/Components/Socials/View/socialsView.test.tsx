import "../../../setupTests";
import React from "react";
import * as Redux from "react-redux";
import { render } from "@testing-library/react";
import { ContentPageData } from "../../../Store/Defaults";
import { SocialsView } from "./socialsView";

jest.mock("react-redux", () => ({
    ...jest.requireActual("react-redux"),
    useSelector: jest.fn(),
}));

describe("test component: socialsView", () => {
    const testContent = {
        language: "eng",
        caption: "Social Media",
        social1: {
            images: {
                header: "linkedin_header.webp",
                avatar: "linkedin_user.webp",
                icon: "linkedin",
            },
            textTitle: "Somebody",
            textSubtitle: "Software Engineer...",
            textComment: "Warsaw Metropolitan Area",
            action: {
                text: "Visit",
                href: "https://google.com",
            },
        },
        social2: {
            images: {
                header: "github_header.webp",
                avatar: "github_user.webp",
                icon: "github",
            },
            textTitle: "Somebody",
            textSubtitle: "JavaScript/TypeScript...",
            textComment: "GitHub Repository",
            action: {
                text: "Visit",
                href: "https://google.com",
            },
        },
        social3: {
            images: {
                header: "instagram_header.webp",
                avatar: "instagram_user.webp",
                icon: "instagram",
            },
            textTitle: "TomKandula",
            textSubtitle: "Photographer ðŸ“¸ ...",
            textComment: "Personal photo blog",
            action: {
                text: "Visit",
                href: "https://google.com",
            },
        },
    };

    const pageData = ContentPageData;
    pageData.components.sectionSocials = testContent;

    beforeEach(() => {
        jest.spyOn(Redux, "useSelector").mockReturnValueOnce(pageData);
    });

    it("should render correctly '<SocialsView />' when content is loaded.", () => {
        const html = render(<SocialsView />);
        expect(html).toMatchSnapshot();
    });
});
