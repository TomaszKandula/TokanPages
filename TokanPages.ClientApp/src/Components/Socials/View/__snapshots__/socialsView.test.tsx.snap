// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component: socialsView should render correctly '<SocialsView />' when content is loaded. 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section>
        <div
          class="bulma-container"
        >
          <div
            class="py-6"
          >
            <div
              data-aos="fade-down"
              data-aos-delay="0"
            >
              <p
                class="is-size-3	has-text-centered has-text-link"
              >
                SOCIAL MEDIA
              </p>
            </div>
            <div
              class="bulma-columns m-6"
            >
              <div
                class="bulma-column is-clickable "
              >
                <div
                  data-aos="fade-up"
                  data-aos-delay="350"
                >
                  <div
                    class="bulma-card"
                  >
                    <a
                      class="href undefined"
                      href="https://google.com"
                      rel="noopener nofollow"
                      target="_blank"
                    >
                      <div
                        class="bulma-card-image"
                      >
                        <figure
                          class="bulma-image"
                        >
                          <img
                            alt="Somebody"
                            class="socials-image lazyloaded"
                            loading="lazy"
                            src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/socials/linkedin_header.webp"
                            title="Illustration"
                          />
                        </figure>
                      </div>
                      <div
                        class="socials-card-image-holder"
                      >
                        <img
                          alt="Somebody"
                          class="socials-avatar lazyloaded"
                          loading="lazy"
                          src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/socials/linkedin_user.webp"
                          title="Socials"
                        />
                      </div>
                      <div
                        class="bulma-card-content pt-4 pb-6"
                      >
                        <div
                          class="is-flex is-justify-content-center pt-6"
                        >
                          <figure
                            class="bulma-image bulma-is-32x32"
                          >
                            <svg
                              fill="#000000"
                              height="2rem"
                              version="1.1"
                              viewBox="0 0 32 32"
                              width="2rem"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M28.778 1.004h-25.56c-0.008-0-0.017-0-0.027-0-1.199 0-2.172 0.964-2.186 2.159v25.672c0.014 1.196 0.987 2.161 2.186 2.161 0.010 0 0.019-0 0.029-0h25.555c0.008 0 0.018 0 0.028 0 1.2 0 2.175-0.963 2.194-2.159l0-0.002v-25.67c-0.019-1.197-0.994-2.161-2.195-2.161-0.010 0-0.019 0-0.029 0h0.001zM9.9 26.562h-4.454v-14.311h4.454zM7.674 10.293c-1.425 0-2.579-1.155-2.579-2.579s1.155-2.579 2.579-2.579c1.424 0 2.579 1.154 2.579 2.578v0c0 0.001 0 0.002 0 0.004 0 1.423-1.154 2.577-2.577 2.577-0.001 0-0.002 0-0.003 0h0zM26.556 26.562h-4.441v-6.959c0-1.66-0.034-3.795-2.314-3.795-2.316 0-2.669 1.806-2.669 3.673v7.082h-4.441v-14.311h4.266v1.951h0.058c0.828-1.395 2.326-2.315 4.039-2.315 0.061 0 0.121 0.001 0.181 0.003l-0.009-0c4.5 0 5.332 2.962 5.332 6.817v7.855z"
                              />
                            </svg>
                          </figure>
                        </div>
                        <div
                          class="is-size-5 has-text-dark has-text-weight-semibold has-text-centered py-2"
                        >
                          Somebody
                        </div>
                        <div
                          class="is-align-content-center socials-text"
                        >
                          <h3
                            class="is-size-6 has-text-grey has-text-centered py-2"
                          >
                            Software Engineer...
                          </h3>
                        </div>
                        <h4
                          class="is-size-6 has-text-dark has-text-weight-semibold has-text-centered py-2"
                        >
                          Warsaw Metropolitan Area
                        </h4>
                      </div>
                    </a>
                  </div>
                </div>
              </div>
              <div
                class="bulma-column is-clickable "
              >
                <div
                  data-aos="fade-up"
                  data-aos-delay="150"
                >
                  <div
                    class="bulma-card"
                  >
                    <a
                      class="href undefined"
                      href="https://google.com"
                      rel="noopener nofollow"
                      target="_blank"
                    >
                      <div
                        class="bulma-card-image"
                      >
                        <figure
                          class="bulma-image"
                        >
                          <img
                            alt="Somebody"
                            class="socials-image lazyloaded"
                            loading="lazy"
                            src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/socials/github_header.webp"
                            title="Illustration"
                          />
                        </figure>
                      </div>
                      <div
                        class="socials-card-image-holder"
                      >
                        <img
                          alt="Somebody"
                          class="socials-avatar lazyloaded"
                          loading="lazy"
                          src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/socials/github_user.webp"
                          title="Socials"
                        />
                      </div>
                      <div
                        class="bulma-card-content pt-4 pb-6"
                      >
                        <div
                          class="is-flex is-justify-content-center pt-6"
                        >
                          <figure
                            class="bulma-image bulma-is-32x32"
                          >
                            <svg
                              fill="none"
                              height="2rem"
                              viewBox="0 0 48 48"
                              width="2rem"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <circle
                                cx="24"
                                cy="24"
                                fill="#181717"
                                r="20"
                              />
                              <path
                                d="M6.81348 34.235C9.24811 38.3138 13.0939 41.4526 17.6772 42.9784C18.6779 43.1614 19.0425 42.5438 19.0425 42.0134C19.0425 41.7938 19.0388 41.4058 19.0339 40.8866C19.0282 40.2852 19.0208 39.5079 19.0155 38.6124C13.4524 39.8206 12.2787 35.931 12.2787 35.931C11.3689 33.6215 10.0576 33.0064 10.0576 33.0064C8.2417 31.7651 10.1951 31.7896 10.1951 31.7896C12.2025 31.9321 13.2584 33.8511 13.2584 33.8511C15.0424 36.9071 17.94 36.0243 19.0794 35.5135C19.2611 34.2207 19.7767 33.3391 20.3489 32.8394C15.908 32.3348 11.2387 30.6183 11.2387 22.9545C11.2387 20.7715 12.0184 18.9863 13.2977 17.5879C13.0914 17.082 12.4051 15.0488 13.4929 12.2949C13.4929 12.2949 15.1725 11.7571 18.9934 14.3453C20.5883 13.9021 22.2998 13.6798 24.0003 13.6725C25.6983 13.6798 27.4099 13.9021 29.0072 14.3453C32.8256 11.7571 34.5016 12.2949 34.5016 12.2949C35.5931 15.0488 34.9067 17.082 34.7005 17.5879C35.9823 18.9863 36.757 20.7715 36.757 22.9545C36.757 30.638 32.0804 32.3286 27.6247 32.8234C28.343 33.441 28.9827 34.6614 28.9827 36.5277C28.9827 38.3152 28.9717 39.8722 28.9644 40.9035C28.9608 41.4143 28.9581 41.7962 28.9581 42.0134C28.9581 42.5487 29.3178 43.1712 30.3332 42.976C33.9844 41.7572 37.1671 39.5154 39.5403 36.5903C35.8734 41.1108 30.274 44 23.9997 44C16.6943 44 10.3038 40.0832 6.81348 34.235Z"
                                fill="white"
                              />
                            </svg>
                          </figure>
                        </div>
                        <div
                          class="is-size-5 has-text-dark has-text-weight-semibold has-text-centered py-2"
                        >
                          Somebody
                        </div>
                        <div
                          class="is-align-content-center socials-text"
                        >
                          <h3
                            class="is-size-6 has-text-grey has-text-centered py-2"
                          >
                            JavaScript/TypeScript...
                          </h3>
                        </div>
                        <h4
                          class="is-size-6 has-text-dark has-text-weight-semibold has-text-centered py-2"
                        >
                          GitHub Repository
                        </h4>
                      </div>
                    </a>
                  </div>
                </div>
              </div>
              <div
                class="bulma-column is-clickable "
              >
                <div
                  data-aos="fade-up"
                  data-aos-delay="250"
                >
                  <div
                    class="bulma-card"
                  >
                    <a
                      class="href undefined"
                      href="https://google.com"
                      rel="noopener nofollow"
                      target="_blank"
                    >
                      <div
                        class="bulma-card-image"
                      >
                        <figure
                          class="bulma-image"
                        >
                          <img
                            alt="TomKandula"
                            class="socials-image lazyloaded"
                            loading="lazy"
                            src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/socials/instagram_header.webp"
                            title="Illustration"
                          />
                        </figure>
                      </div>
                      <div
                        class="socials-card-image-holder"
                      >
                        <img
                          alt="TomKandula"
                          class="socials-avatar lazyloaded"
                          loading="lazy"
                          src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/socials/instagram_user.webp"
                          title="Socials"
                        />
                      </div>
                      <div
                        class="bulma-card-content pt-4 pb-6"
                      >
                        <div
                          class="is-flex is-justify-content-center pt-6"
                        >
                          <figure
                            class="bulma-image bulma-is-32x32"
                          >
                            <svg
                              fill="none"
                              height="2rem"
                              viewBox="0 0 24 24"
                              width="2rem"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                clip-rule="evenodd"
                                d="M2 6C2 3.79086 3.79086 2 6 2H18C20.2091 2 22 3.79086 22 6V18C22 20.2091 20.2091 22 18 22H6C3.79086 22 2 20.2091 2 18V6ZM6 4C4.89543 4 4 4.89543 4 6V18C4 19.1046 4.89543 20 6 20H18C19.1046 20 20 19.1046 20 18V6C20 4.89543 19.1046 4 18 4H6ZM12 9C10.3431 9 9 10.3431 9 12C9 13.6569 10.3431 15 12 15C13.6569 15 15 13.6569 15 12C15 10.3431 13.6569 9 12 9ZM7 12C7 9.23858 9.23858 7 12 7C14.7614 7 17 9.23858 17 12C17 14.7614 14.7614 17 12 17C9.23858 17 7 14.7614 7 12ZM17.5 8C18.3284 8 19 7.32843 19 6.5C19 5.67157 18.3284 5 17.5 5C16.6716 5 16 5.67157 16 6.5C16 7.32843 16.6716 8 17.5 8Z"
                                fill="#000000"
                                fill-rule="evenodd"
                              />
                            </svg>
                          </figure>
                        </div>
                        <div
                          class="is-size-5 has-text-dark has-text-weight-semibold has-text-centered py-2"
                        >
                          TomKandula
                        </div>
                        <div
                          class="is-align-content-center socials-text"
                        >
                          <h3
                            class="is-size-6 has-text-grey has-text-centered py-2"
                          >
                            Photographer 📸 ...
                          </h3>
                        </div>
                        <h4
                          class="is-size-6 has-text-dark has-text-weight-semibold has-text-centered py-2"
                        >
                          Personal photo blog
                        </h4>
                      </div>
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section>
      <div
        class="bulma-container"
      >
        <div
          class="py-6"
        >
          <div
            data-aos="fade-down"
            data-aos-delay="0"
          >
            <p
              class="is-size-3	has-text-centered has-text-link"
            >
              SOCIAL MEDIA
            </p>
          </div>
          <div
            class="bulma-columns m-6"
          >
            <div
              class="bulma-column is-clickable "
            >
              <div
                data-aos="fade-up"
                data-aos-delay="350"
              >
                <div
                  class="bulma-card"
                >
                  <a
                    class="href undefined"
                    href="https://google.com"
                    rel="noopener nofollow"
                    target="_blank"
                  >
                    <div
                      class="bulma-card-image"
                    >
                      <figure
                        class="bulma-image"
                      >
                        <img
                          alt="Somebody"
                          class="socials-image lazyloaded"
                          loading="lazy"
                          src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/socials/linkedin_header.webp"
                          title="Illustration"
                        />
                      </figure>
                    </div>
                    <div
                      class="socials-card-image-holder"
                    >
                      <img
                        alt="Somebody"
                        class="socials-avatar lazyloaded"
                        loading="lazy"
                        src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/socials/linkedin_user.webp"
                        title="Socials"
                      />
                    </div>
                    <div
                      class="bulma-card-content pt-4 pb-6"
                    >
                      <div
                        class="is-flex is-justify-content-center pt-6"
                      >
                        <figure
                          class="bulma-image bulma-is-32x32"
                        >
                          <svg
                            fill="#000000"
                            height="2rem"
                            version="1.1"
                            viewBox="0 0 32 32"
                            width="2rem"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M28.778 1.004h-25.56c-0.008-0-0.017-0-0.027-0-1.199 0-2.172 0.964-2.186 2.159v25.672c0.014 1.196 0.987 2.161 2.186 2.161 0.010 0 0.019-0 0.029-0h25.555c0.008 0 0.018 0 0.028 0 1.2 0 2.175-0.963 2.194-2.159l0-0.002v-25.67c-0.019-1.197-0.994-2.161-2.195-2.161-0.010 0-0.019 0-0.029 0h0.001zM9.9 26.562h-4.454v-14.311h4.454zM7.674 10.293c-1.425 0-2.579-1.155-2.579-2.579s1.155-2.579 2.579-2.579c1.424 0 2.579 1.154 2.579 2.578v0c0 0.001 0 0.002 0 0.004 0 1.423-1.154 2.577-2.577 2.577-0.001 0-0.002 0-0.003 0h0zM26.556 26.562h-4.441v-6.959c0-1.66-0.034-3.795-2.314-3.795-2.316 0-2.669 1.806-2.669 3.673v7.082h-4.441v-14.311h4.266v1.951h0.058c0.828-1.395 2.326-2.315 4.039-2.315 0.061 0 0.121 0.001 0.181 0.003l-0.009-0c4.5 0 5.332 2.962 5.332 6.817v7.855z"
                            />
                          </svg>
                        </figure>
                      </div>
                      <div
                        class="is-size-5 has-text-dark has-text-weight-semibold has-text-centered py-2"
                      >
                        Somebody
                      </div>
                      <div
                        class="is-align-content-center socials-text"
                      >
                        <h3
                          class="is-size-6 has-text-grey has-text-centered py-2"
                        >
                          Software Engineer...
                        </h3>
                      </div>
                      <h4
                        class="is-size-6 has-text-dark has-text-weight-semibold has-text-centered py-2"
                      >
                        Warsaw Metropolitan Area
                      </h4>
                    </div>
                  </a>
                </div>
              </div>
            </div>
            <div
              class="bulma-column is-clickable "
            >
              <div
                data-aos="fade-up"
                data-aos-delay="150"
              >
                <div
                  class="bulma-card"
                >
                  <a
                    class="href undefined"
                    href="https://google.com"
                    rel="noopener nofollow"
                    target="_blank"
                  >
                    <div
                      class="bulma-card-image"
                    >
                      <figure
                        class="bulma-image"
                      >
                        <img
                          alt="Somebody"
                          class="socials-image lazyloaded"
                          loading="lazy"
                          src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/socials/github_header.webp"
                          title="Illustration"
                        />
                      </figure>
                    </div>
                    <div
                      class="socials-card-image-holder"
                    >
                      <img
                        alt="Somebody"
                        class="socials-avatar lazyloaded"
                        loading="lazy"
                        src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/socials/github_user.webp"
                        title="Socials"
                      />
                    </div>
                    <div
                      class="bulma-card-content pt-4 pb-6"
                    >
                      <div
                        class="is-flex is-justify-content-center pt-6"
                      >
                        <figure
                          class="bulma-image bulma-is-32x32"
                        >
                          <svg
                            fill="none"
                            height="2rem"
                            viewBox="0 0 48 48"
                            width="2rem"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <circle
                              cx="24"
                              cy="24"
                              fill="#181717"
                              r="20"
                            />
                            <path
                              d="M6.81348 34.235C9.24811 38.3138 13.0939 41.4526 17.6772 42.9784C18.6779 43.1614 19.0425 42.5438 19.0425 42.0134C19.0425 41.7938 19.0388 41.4058 19.0339 40.8866C19.0282 40.2852 19.0208 39.5079 19.0155 38.6124C13.4524 39.8206 12.2787 35.931 12.2787 35.931C11.3689 33.6215 10.0576 33.0064 10.0576 33.0064C8.2417 31.7651 10.1951 31.7896 10.1951 31.7896C12.2025 31.9321 13.2584 33.8511 13.2584 33.8511C15.0424 36.9071 17.94 36.0243 19.0794 35.5135C19.2611 34.2207 19.7767 33.3391 20.3489 32.8394C15.908 32.3348 11.2387 30.6183 11.2387 22.9545C11.2387 20.7715 12.0184 18.9863 13.2977 17.5879C13.0914 17.082 12.4051 15.0488 13.4929 12.2949C13.4929 12.2949 15.1725 11.7571 18.9934 14.3453C20.5883 13.9021 22.2998 13.6798 24.0003 13.6725C25.6983 13.6798 27.4099 13.9021 29.0072 14.3453C32.8256 11.7571 34.5016 12.2949 34.5016 12.2949C35.5931 15.0488 34.9067 17.082 34.7005 17.5879C35.9823 18.9863 36.757 20.7715 36.757 22.9545C36.757 30.638 32.0804 32.3286 27.6247 32.8234C28.343 33.441 28.9827 34.6614 28.9827 36.5277C28.9827 38.3152 28.9717 39.8722 28.9644 40.9035C28.9608 41.4143 28.9581 41.7962 28.9581 42.0134C28.9581 42.5487 29.3178 43.1712 30.3332 42.976C33.9844 41.7572 37.1671 39.5154 39.5403 36.5903C35.8734 41.1108 30.274 44 23.9997 44C16.6943 44 10.3038 40.0832 6.81348 34.235Z"
                              fill="white"
                            />
                          </svg>
                        </figure>
                      </div>
                      <div
                        class="is-size-5 has-text-dark has-text-weight-semibold has-text-centered py-2"
                      >
                        Somebody
                      </div>
                      <div
                        class="is-align-content-center socials-text"
                      >
                        <h3
                          class="is-size-6 has-text-grey has-text-centered py-2"
                        >
                          JavaScript/TypeScript...
                        </h3>
                      </div>
                      <h4
                        class="is-size-6 has-text-dark has-text-weight-semibold has-text-centered py-2"
                      >
                        GitHub Repository
                      </h4>
                    </div>
                  </a>
                </div>
              </div>
            </div>
            <div
              class="bulma-column is-clickable "
            >
              <div
                data-aos="fade-up"
                data-aos-delay="250"
              >
                <div
                  class="bulma-card"
                >
                  <a
                    class="href undefined"
                    href="https://google.com"
                    rel="noopener nofollow"
                    target="_blank"
                  >
                    <div
                      class="bulma-card-image"
                    >
                      <figure
                        class="bulma-image"
                      >
                        <img
                          alt="TomKandula"
                          class="socials-image lazyloaded"
                          loading="lazy"
                          src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/socials/instagram_header.webp"
                          title="Illustration"
                        />
                      </figure>
                    </div>
                    <div
                      class="socials-card-image-holder"
                    >
                      <img
                        alt="TomKandula"
                        class="socials-avatar lazyloaded"
                        loading="lazy"
                        src="http://localhost:5000/api/v1/content/assets/getNonVideoAsset?blobName=images/sections/socials/instagram_user.webp"
                        title="Socials"
                      />
                    </div>
                    <div
                      class="bulma-card-content pt-4 pb-6"
                    >
                      <div
                        class="is-flex is-justify-content-center pt-6"
                      >
                        <figure
                          class="bulma-image bulma-is-32x32"
                        >
                          <svg
                            fill="none"
                            height="2rem"
                            viewBox="0 0 24 24"
                            width="2rem"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              clip-rule="evenodd"
                              d="M2 6C2 3.79086 3.79086 2 6 2H18C20.2091 2 22 3.79086 22 6V18C22 20.2091 20.2091 22 18 22H6C3.79086 22 2 20.2091 2 18V6ZM6 4C4.89543 4 4 4.89543 4 6V18C4 19.1046 4.89543 20 6 20H18C19.1046 20 20 19.1046 20 18V6C20 4.89543 19.1046 4 18 4H6ZM12 9C10.3431 9 9 10.3431 9 12C9 13.6569 10.3431 15 12 15C13.6569 15 15 13.6569 15 12C15 10.3431 13.6569 9 12 9ZM7 12C7 9.23858 9.23858 7 12 7C14.7614 7 17 9.23858 17 12C17 14.7614 14.7614 17 12 17C9.23858 17 7 14.7614 7 12ZM17.5 8C18.3284 8 19 7.32843 19 6.5C19 5.67157 18.3284 5 17.5 5C16.6716 5 16 5.67157 16 6.5C16 7.32843 16.6716 8 17.5 8Z"
                              fill="#000000"
                              fill-rule="evenodd"
                            />
                          </svg>
                        </figure>
                      </div>
                      <div
                        class="is-size-5 has-text-dark has-text-weight-semibold has-text-centered py-2"
                      >
                        TomKandula
                      </div>
                      <div
                        class="is-align-content-center socials-text"
                      >
                        <h3
                          class="is-size-6 has-text-grey has-text-centered py-2"
                        >
                          Photographer 📸 ...
                        </h3>
                      </div>
                      <h4
                        class="is-size-6 has-text-dark has-text-weight-semibold has-text-centered py-2"
                      >
                        Personal photo blog
                      </h4>
                    </div>
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
